packages/shorebird_cli/lib/src/code_push_client_wrapper.dart

```diff
@@ -162,7 +163,10 @@ This app may not exist or you may not have permission to view it.''',
       logger.err(
         '''
 It looks like you have an existing ${platform.name} release for version ${lightCyan.wrap(release.version)}.
-Please bump your version number and try again.''',
+Please bump your version number and try again.
+
+You can manage this release in the Shorebird Console at:
+${ShorebirdWebConsole.linkToAppRelease(release.appId, release.id)}''',
```

Comment:
You probably want to wrap this with `link` from `mason_logger` so that it can be easily clicked on in multiple terminals to open the link externally.

https://pub.dev/documentation/mason_logger/latest/mason_logger/link.html